datasets/fce/fce.py

```diff
@@ -0,0 +1,130 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+
+
+import datalabs
+from datalabs import get_task, TaskType
+import os
+
+logger = datalabs.logging.get_logger(__name__)
+
+_CITATION = """\
+@inproceedings{yannakoudakis2011new,
+  title={A new dataset and method for automatically grading ESOL texts},
+  author={Yannakoudakis, Helen and Briscoe, Ted and Medlock, Ben},
+  booktitle={Proceedings of the 49th annual meeting of the association for computational linguistics: human language technologies},
+  pages={180--189},
+  year={2011}
+}
+"""
+
+_DESCRIPTION = """\
+The Cambridge Learner Corpus First Certificate in English (CLC FCE) dataset consists of short texts, written by learners of English as an additional language in response to exam prompts eliciting free-text answers and assessing mastery of the upper-intermediate proficiency level. The texts have been manually error-annotated using a taxonomy of 77 error types. The full dataset consists of 323,192 sentences. The publicly released subset of the dataset, named FCE-public, consists of 33,673 sentences split into test and training sets of 2,720 and 30,953 sentences, respectively.
+"""
+
+_URLS = "https://datalab-hub.s3.amazonaws.com/grammatical_error_correction/fce/fce.zip"
+
+
+class FceConfig(datalabs.BuilderConfig):
+    """BuilderConfig for fce."""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for fce.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(FceConfig, self).__init__(**kwargs)
+
+
+class Fce(datalabs.GeneratorBasedBuilder):
+
+    BUILDER_CONFIGS = [
+        FceConfig(
+            name="plain_text",
+            version=datalabs.Version("2.1.0", ""),
+            description="Plain text",
+        ),
+    ]
+
+    def _info(self):
+        return datalabs.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datalabs.Features(
+                {
+                    "original": datalabs.Value("string"),
+                    "correct":
+                        {
+                            "m2": datalabs.features.Sequence(datalabs.Value("string")),
+                            "sentence": datalabs.Value("string"),
+                        }
+
+                }
+            ),
+            supervised_keys=None,
+            homepage="https://ilexir.co.uk/datasets/index.html",
+            citation=_CITATION,
+            task_templates=[
+                get_task(TaskType.grammatical_error_correction)(
```

Comment:
There has been the `grammatical-error-correction` task: 
https://github.com/ExpressAI/DataLab/blob/55e7d79379ecf4cf534846cd121adc0f8a43f78b/datalabs/tasks/text_editing.py#L41  we should figure out what the differences are